libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/Diagnostic.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+using Antlr4.Runtime;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Parser
+{
+    public enum DiagnosticSeverity
```

Comment:
The ENUM is fragile, as over time you're gong to want to add "Info" "Verbose", "None" and others. You won't be able to do that w/o breaking compat. Consider eliminateing this and just using strings.

Reply:
Solved